pytest_variables.py

```diff
@@ -14,14 +27,20 @@ def pytest_addoption(parser):
         action='append',
         default=[],
         metavar='path',
-        help='path to test variables JSON file.')
+        help='path to test variables JSON/HJSON/YAML file.')
```

Comment:
I wonder if we should allow the `--variables` argument to be two values. If the second if specified it explicitly sets the format, but if it's not we default to using the extension. Such as: `--variables vars.txt yaml`. This would be useful if the test runner does not have the ability to set the extension, such as Jenkins using the Config File Provider plugin. Note that this could also be a future enhancement.